### ISSUE TITLE
level selector not aligned

### ISSUE THREAD

**Issue by Niko78** [clappr/clappr-level-selector-plugin#49]
I noticed that level selector (last one with last clappr) is not aligned in height like other (volume, fullscreen)
![quality](https://cloud.githubusercontent.com/assets/17108017/23223163/940aeefc-f92a-11e6-9a03-06328d269aef.png)

**Comment by vuhanguyen** [comment]
I got this too, is it a bug?

**Comment by Niko78** [comment]
well it seems issues in scss I suppose
+ it's also like menu when I click on it, it seems very small in larger maybe a bit large would be better
<img width="196" alt="bar" src="https://cloud.githubusercontent.com/assets/17108017/23260281/4d75d5b8-f9d1-11e6-9f35-8b41e5bae5cc.png">

**Comment by vuhanguyen** [comment]
Yes that's right. It's affected by CSS style in my page, I have to override the style to make it displayed on right place (line-height:32px, padding: 0!important )

**Comment by Niko78** [comment]
you should "pull reuqest" in changing code
level selector could be more "design" (but I'm not designer)
+ it's mixed with clappr HD icon
